examples/line-liff-v1/.env

```diff
@@ -0,0 +1,16 @@
+MESSENGER_PAGE_ID=
+MESSENGER_ACCESS_TOKEN=
+MESSENGER_APP_ID=
+MESSENGER_APP_SECRET=
+MESSENGER_VERIFY_TOKEN=
```

Comment:
To provide a minimal example, we can remove those non-line env just like what we did in other examples:

https://github.com/Yoctol/bottender/blob/master/examples/line-hello-world/.env